dedibox-network/rpn/how-to/use-rpn-san-esxi-v7.mdx

```diff
@@ -0,0 +1,208 @@
+
+2. Click **Actions** > **Services** > **Enable Secure Shell (SSH)**:
+
+  <Lightbox src="scaleway-enable_ssh.webp" width="600" alt="" />
+
+  The following warning displays once SSH is enabled: **SSH is enabled on this host. You should disable SSH unless it is necessary for administrative purposes.**
+
+4. Create a new **Virtual Switch** by clicking on **Networking** > **Virtual Switches** > **Add Standard Virtual Switch**. Configure it as shown in the following example:
+
+  <Lightbox src="scaleway-rpn_switch.webp" width="400" alt="" />
+
+5. Create new **VMkernel NIC** by clicking on **Networking** > **VMkernel NICs** > **Add VMkernel NIC** and configure it as follows. 
+
+  <Lightbox src="scaleway-vmkernel_nic.webp" width="400" alt="" />
+
+  <Message type="note">
+    
+    Make sure to configure Jumboframes (MTU 9000) both for the switch and the NIC.
+
+  </Message>
+
+6. Connect to your server using SSH. The SSH credentials are identical to those for the web interface.
+
+7. Locate the MAC address of the network card connected to the RPN (MTU 9000):
+  ```
+  esxcfg-nics -l
+  ```
+
+8. Export the current VMkernel NIC configuration:
+  ```
+  configstorecli config current get -c esx -g network -k vmknics > vmknics.json
+  ```
+
+9. Open the VMkernel NIC configuration file in a text editor:
+
+  ```
+  vi vmknics.json
+  ```
+
+10. Replace the MAC address of your RPN VMkernel NIC with your [virtual MAC](/dedibox-network/ip-failover/how-to/create-virtual-mac).
+
+  <Lightbox src="scaleway-esxi7-vmnic.webp" />
+
+11. Import the new configuration:
+
+  ```
+  configstorecli config current set -c esx -g network -k vmknics -i vmknics.json --overwrite
+  ```
+
+12. Reboot the machine to activate the new configuration.
+
+### Setting RPN routes
+
+1. Enter the ESXi web interface, click **Networking** > **Virtual Switches** > **RPN Switch** and check the IP address of the VMkernel NIC. It now has the RPN IP auto assigned to the interface via DHCP:
+
+  <Lightbox src="scaleway-rpn_switch_connected.webp" width="800" alt="" />
+
+2. Reactivte the SSH service and connect to the machine via SSH.
+
+3. Run the following command to add the RPN route:
+
+  ```
+  esxcfg-route -a 10.88.0.0/13 [YOUR_RPN_GATEWAY_IP]
+  ```
+
+  <Message type="tip"> 
+
+  You can find the RPN gateway either on your server's overview page in the Dedibox console or by following [this guide](/dedibox-network/rpn/how-to/find-rpn-gateway/).
+
+  </Message>
+
+4. Ping the RPN-SAN to check if the RPN connection is working properly:
+
+  ```
+  ping san-1.rpn.online.net
+  ```
+
+5. Open the file `/etc/rc.local.d/local.sh` in a text editor and add the following lines to it to configure the RPN route automatically during boot:
+
+  ```
+  esxcfg-route -a 10.88.0.0/13 [YOUR_RPN_GATEWAY_IP]
+  exit 0
+  ```
+
+### Connecting the RPN-SAN
+
+<Message type="note">
```

Comment:
I don't think this should be in a note, it should just be the introduction to this section (normal text)